### DISCUSSION TITLE
Using oh-my-zsh to version function with the git dir

### DISCUSSION THREAD

**Discussion by TunnelThruTime** [ohmyzsh/ohmyzsh#12223] [Help] [answered]
How do you version functions with oh-my-zsh? If I stored my custom functions in the oh-my-zsh repo would that affect the updates from omz? How do ppl how version functions work with omz?

**Comment by carlosala** [comment, marked as answer]
Hi! To store your own stuff, you can use `custom/` directory https://github.com/ohmyzsh/ohmyzsh?tab=readme-ov-file#custom-plugins-and-themes
You can update to latest version (we live on `master`, no versioning at all for the moment) running `omz update` (it'll not break your modifications in `custom/` directory).
